.circleci/config.yml

```diff
@@ -95,8 +95,8 @@ jobs:
           name: Run flake8
           command: |
             . venv/bin/activate
-            flake8 --ignore E501 --exclude onecodex/schemas/* onecodex/
-            flake8 --ignore E501 --exclude onecodex/schemas/* tests/
+            flake8 --ignore E501,W504,W605 --exclude onecodex/schemas/* onecodex/
```

Comment:
Let's fix these linting issues rather than adding an ignore. Fix for `W605` is to use raw strings, e.g., `rb'\d'`. Fix for `W504` is to probably rewrite some of those lines with `.format()` which is better style anyhow.

`W605` is important in particular as it looks like this behavior will break Python 3.8 once that's released.

Reply:
Okey dokey, I re-enabled the warnings and fixed them. W605 was mostly adding an additional escape character--adding 'r' or 'rb' seemed to break things. W504 was mostly changing the way long strings were split across multiple lines.